server/src/test/java/org/elasticsearch/plugins/PluginsServiceTests.java

```diff
@@ -531,6 +531,10 @@ public TestExtension() {}
             public TestExtension(TestPlugin plugin) {
 
             }
+
+            public TestExtension(TestPlugin plugin, String anotherArg) {
```

Comment:
Is this used somewhere?

Reply:
Yes, I added a constructor to make the test check that we disallow a service provider with more than 2 constructors. Previously this class was testing for existence of more than 1.